### PR TITLE
Artist revolvers normalization and old rocket addition

### PR DESCRIPTION
**Pull request by Iglpigl**
## About The Pull Request

Plastic cap revolver has been removed because of it not working as it was supposed to and its ammo being extremely rare, RPG and AMR revolver were re-added, RPGs ammo is only obtainable from serbs, while AMR no longer stuns and it's damage variables are tweaked correctly, recoil has also been changed on all artist revolver versions to take into account recoil rework, and considering source guns that were used for revolver damage and AP were changed those values were updated too, some numbers were increased or reduced, also makes old rockets craftable via crafting bench.

## Why It's Good For The Game

Outdated numbers being fixed is good, also the AMR and RPG revolver deserve to exist, especially now with all balance concerns addressed, and plastic cap revolver doesn't work as it should and it honestly looks like it was randomly added just because it would fit.

## Changelog
:cl:
add: Adds old rockets to the things you can build on the crafting station; Re-added AMR and RPG revolvers with correct numbers
del: Removed plastic cap revolver from possible revolvers artist can make
tweak: tweaked recoil, damage, AP, one hand penalty of all artist revolvers to take into account recoil rework and other variable changes that happened in the meantime
spellcheck: Fixes minor spelling mistake when crafting station was full, also makes normal RPG ammo description more organic
/:cl:

### CODE REVIEW

**Comment by zombiemimic** [comment]
Don't re-add the RPG revolver unless you intend it to be usable, please

**Comment by Wouju** [comment]
RPG revolver is peak of Eris guns, having it exist is a must.

**Comment by Iglpigl** [comment]
> Don't re-add the RPG revolver unless you intend it to be usable, please

It's usable, it's ammo might even be possible to find on blacksights, haven't checked.

**Comment by Kegdo** [comment]
Let's be real, nobody goes on blacksights. Maybe add RPG ammo to one of the trading beacons?

**Comment by TheShown911** [comment]
No. Just make weaker RPG shells craftable.

**Comment by Sparrowbird611** [comment]
Adding the RPG revolver is really unnecessary and shouldn't be brought back. AMR is probably alright to see back, though I would still have some concerns. Cap gun removal is gud.

**Comment by Anonmare** [comment]
image

**Comment by tristain01** [comment]
I hear that artist attacks with all kinds of random bullshit.

File: code/game/machinery/autolathe/artist_bench.dm [hunk 1]
```diff
@@ -133,43 +133,56 @@
 			"magnum" = 8 + weight_vigilance,
 			"shotgun" = 8 + weight_robustness,
 			"rifle" = 8 + weight_vigilance,
-			"cap" = 16 + weight_biology,
+			"sniper" = 8 + max(weight_vigilance + weight_cognition),
```
Comment (Evankhell561) [inline code/game/machinery/autolathe/artist_bench.dm:136]:
```suggestion
			"sniper" = 8 + max(weight_vigilance, weight_cognition),
```

**Review by Evankhell561 — CHANGES_REQUESTED** [review]

File: code/game/machinery/crafting_station.dm [hunk 2]
```diff
@@ -444,6 +456,21 @@
 
 //////////////////////////////////////////////////////////////////////////////////////////////
 
+/obj/machinery/craftingstation/proc/spawn_rocket(dice = 0, mob/user)	//All rifles use same spawning stats
+
+	var/piles = 0
```
Comment (Evankhell561) [inline code/game/machinery/crafting_station.dm:461]:
```suggestion
	var/piles
```

**Review by Evankhell561 — DISMISSED** [review]

File: code/game/machinery/autolathe/artist_bench.dm [hunk 3]
```diff
@@ -133,43 +133,60 @@
...
-				R.recoil_buildup = 18 + rand(-3,3)
+				R.damage_multiplier = 1.4 + rand(-5,5)/10
+				R.penetration_multiplier = 1.4 + rand(-5,5)/10
+				R.recoil_buildup = 3 + rand(-3,3)
 
 			if("magnum") //From consul.dm, Arbitrary values
 				R.caliber = CAL_MAGNUM
-				R.damage_multiplier = 1.2 + rand(-5,5)/10
-				R.penetration_multiplier = 1.2 + rand(-5,5)/10
-				R.recoil_buildup = 35 + rand(-5,5)
```
Comment (TheShown911) [inline code/game/machinery/autolathe/artist_bench.dm:151]: If you change the base recoil also change the amount randomized. 35 +- 5 is not much, but 6 +- 5 is a lot

File: code/modules/projectiles/ammunition/bullets.dm [hunk 4]
```diff
@@ -468,6 +468,20 @@
...
+/obj/item/ammo_casing/rocket/scrap
+	name = "Scrap rocket"
+	desc = "An old improvised rocket shell for the RPG-7 launcher. Has a tubular shape."
+	icon_state = "old_rocketshell"
+	projectile_type = /obj/item/projectile/bullet/rocket/scrap
+	caliber = CAL_ROCKET
+	maxamount = 1
+	reload_delay = 15
+	is_caseless = TRUE
+	w_class = ITEM_SIZE_NORMAL
```
Comment (TheShown911) [inline code/modules/projectiles/ammunition/bullets.dm:480]: You don't need some of these variables as the scrap rocket inherits them from it's parent.

File: code/modules/projectiles/ammunition/bullets.dm [hunk 5]
```diff
@@ -468,6 +468,16 @@
 	is_caseless = TRUE
 	w_class = ITEM_SIZE_NORMAL
 
+/obj/item/ammo_casing/rocket/scrap
+	name = "Scrap rocket"
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/bullets.dm:472]:
```suggestion
	name = "scrap rocket"
```

**Comment by drexample** [comment]
awful

**Pull request merged**